### PR TITLE
Gk blog exercises

### PR DESCRIPTION
**Pull request by gregkorte**
Added three blog exercises.

### CODE REVIEW

**Comment by stevebrownlee** [comment]
@gregkorte Is this PR ready for review?

**Comment by gregkorte** [comment]
@stevebrownlee Yes, this one is ready for review.

File: 3-modern-javascript-developer/exercises/MJ_PROMISES_BLOG_03.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+> This project is made up of several parts. By creating a branch for each section you will be able to preserve the concepts you have learned and able to reference them in the future. We have mentioned you should be commenting your code, right?
+
+## Requirements
+
+Time to modularize your blog and access your posts from Firebase. At this point you should have a minimum of 5 personal blog posts.
+
+### Part One
+
+1. Install Grunt to at least watch and lint your javascript files.
```
Comment (stevebrownlee) [inline 3-modern-javascript-developer/exercises/MJ_PROMISES_BLOG_03.md]: I'd recommend removing the "at least" here

File: 3-modern-javascript-developer/exercises/MJ_PROMISES_BLOG_03.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Part Two
+
+1. Create two modules
+    - blog_controller.js
+    - blog_factory.js
+1. The blog_controller module should contain all of the DOM interaction.
+1. The blog_factory should contain only calls to Firebase.
+1. Upload the json file you created for the Blog 2 exercise to Firebase.
+1. When the index page loads, get the posts from Firebase via ajax using promises.
```
Comment (stevebrownlee) [inline 3-modern-javascript-developer/exercises/MJ_PROMISES_BLOG_03.md]: "GET the posts from Firebase via an XHR wrapped in a Promise"

File: 4-rich-browser-applications/exercises/RBA_BLOG_04.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+> This project is made up of several parts. By creating a branch for each section you will be able to preserve the concepts you have learned and able to reference them in the future. We have mentioned you should be commenting your code, right?
+
+## Requirements
+
+Time to make your blog fully functional. At this point you should have a minimum of 7 personal blog posts.
+
+### Part One
+
+1. Using the previous controller and factory add CRUD functionality to your blog using restful requests.
```
Comment (stevebrownlee) [inline 4-rich-browser-applications/exercises/RBA_BLOG_04.md]: "using the RESTful Firebase interface (GET, POST, PUT, DELETE)"

File: 4-rich-browser-applications/exercises/RBA_BLOG_04.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    - Remember: Only the factory should access Firebase.
+1. All views for the blog should be utilizing handlebars templates.
```
Comment (stevebrownlee) [inline 4-rich-browser-applications/exercises/RBA_BLOG_04.md]: 1. Create a Handlebars template for every view of your blog.

File: 4-rich-browser-applications/exercises/RBA_BLOG_04.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+### Part Two
+
+1. Add authentication to post, delete, and edit.
```
Comment (stevebrownlee) [inline 4-rich-browser-applications/exercises/RBA_BLOG_04.md]: Add one of the Firebase-provided OAuth authentication mechanisms to...

File: 5-modern-frameworks/exercises/MF_BLOG_05.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+> This project is made up of several parts. By creating a branch for each section you will be able to preserve the concepts you have learned and able to reference them in the future. We have mentioned you should be commenting your code, right?
+
+## Requirements
+
+Time to Angularize your blog and add it to your personal site. At this point you should have a minimum of 10 personal blog posts.
+
+### Part One
+
+1. Angularize your blog.
```
Comment (stevebrownlee) [inline 5-modern-frameworks/exercises/MF_BLOG_05.md]: 1. Convert your blog from Handlebars & vanilla JavaScript to a complete Angular-based application.

**Comment by gregkorte** [comment]
@stevebrownlee Thanks for the review. Changes have been made.

**Comment by brendalong** [comment]
👍

**Comment by brendalong** [comment]
👍

**Pull request merged**